.github/workflows/go-apidiff.yaml

```diff
@@ -0,0 +1,25 @@
+name: go-apidiff
+on: [ pull_request ]
+jobs:
+  go-apidiff:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-go@v3
+      with:
+        go-version-file: go.mod
+
+    - uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cache/go-build
+          ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    - uses: joelanford/go-apidiff@main
```

Comment:
Personal repo?

Reply:
I'm the author :)

Reply:
controller-runtime vouches for me: https://github.com/kubernetes-sigs/controller-runtime/blob/ca01f67d57d64cf1af861cea9e4403a6a599abb8/Makefile#L70